common/src/main/java/bisq/common/file/FileUtils.java

```diff
@@ -262,13 +286,35 @@ public static void makeFile(File file) throws IOException {
         }
     }
 
+    public static FileOutputStream newFileOutputStreamWithPermissions(String name) throws IOException {
+        Path path = Paths.get(name);
+        try {
+            Set<PosixFilePermission> perms = EnumSet.of(PosixFilePermission.OWNER_READ, PosixFilePermission.OWNER_WRITE);
```

Comment:
nitpick: perms can be defined as a constant to avoid repeated initialization and to improve code reusability.